Windows build doesn't work after updating bindgen

After pulling the latest commit to master (c905fb45edf0593003f45430b2ef5b02c148c4ba) on Windows 10 the build fails due to 78 `expected u32, found i32` errors. Haven't tested in other environments yet.

Using `rustc 1.33.0-nightly (ceb251214 2019-01-16)`

```
error[E0308]: mismatched types
  --> core\src\lib.rs:99:14
   |
99 |     Failed = sys::godot_error_GODOT_FAILED,
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:100:19
    |
100 |     Unavailable = sys::godot_error_GODOT_ERR_UNAVAILABLE,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:101:20
    |
101 |     Unconfigured = sys::godot_error_GODOT_ERR_UNCONFIGURED,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:102:19
    |
102 |     Unothorized = sys::godot_error_GODOT_ERR_UNAUTHORIZED,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:103:21
    |
103 |     PrameterRange = sys::godot_error_GODOT_ERR_PARAMETER_RANGE_ERROR,
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:104:19
    |
104 |     OutOfMemory = sys::godot_error_GODOT_ERR_OUT_OF_MEMORY,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:105:20
    |
105 |     FileNotFound = sys::godot_error_GODOT_ERR_FILE_NOT_FOUND,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:106:20
    |
106 |     FileBadDrive = sys::godot_error_GODOT_ERR_FILE_BAD_DRIVE,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:107:19
    |
107 |     FileBadPath = sys::godot_error_GODOT_ERR_FILE_BAD_PATH,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:108:24
    |
108 |     FileNoPermission = sys::godot_error_GODOT_ERR_FILE_NO_PERMISSION,
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:109:24
    |
109 |     FileAlreadyInUse = sys::godot_error_GODOT_ERR_FILE_ALREADY_IN_USE,
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:110:20
    |
110 |     FileCantOpen = sys::godot_error_GODOT_ERR_FILE_CANT_OPEN,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:111:21
    |
111 |     FileCantWrite = sys::godot_error_GODOT_ERR_FILE_CANT_WRITE,
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:112:20
    |
112 |     FileCantRead = sys::godot_error_GODOT_ERR_FILE_CANT_READ,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:113:24
    |
113 |     FileUnrecognized = sys::godot_error_GODOT_ERR_FILE_UNRECOGNIZED,
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:114:19
    |
114 |     FileCorrupt = sys::godot_error_GODOT_ERR_FILE_CORRUPT,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:115:29
    |
115 |     FileMissingDependency = sys::godot_error_GODOT_ERR_FILE_MISSING_DEPENDENCIES,
    |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:116:15
    |
116 |     FileEof = sys::godot_error_GODOT_ERR_FILE_EOF,
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:117:16
    |
117 |     CantOpen = sys::godot_error_GODOT_ERR_CANT_OPEN,
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:118:18
    |
118 |     CantCreate = sys::godot_error_GODOT_ERR_CANT_CREATE,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:119:19
    |
119 |     QueryFailed = sys::godot_error_GODOT_ERR_QUERY_FAILED,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:120:20
    |
120 |     AlreadyInUse = sys::godot_error_GODOT_ERR_ALREADY_IN_USE,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:121:14
    |
121 |     Locked = sys::godot_error_GODOT_ERR_LOCKED,
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:122:15
    |
122 |     TimeOut = sys::godot_error_GODOT_ERR_TIMEOUT,
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:123:19
    |
123 |     CantConnect = sys::godot_error_GODOT_ERR_CANT_CONNECT,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:124:19
    |
124 |     CantResolve = sys::godot_error_GODOT_ERR_CANT_RESOLVE,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:125:23
    |
125 |     ConnectionError = sys::godot_error_GODOT_ERR_CONNECTION_ERROR,
    |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:126:27
    |
126 |     CantAcquireResource = sys::godot_error_GODOT_ERR_CANT_ACQUIRE_RESOURCE,
    |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:127:16
    |
127 |     CantFork = sys::godot_error_GODOT_ERR_CANT_FORK,
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:128:19
    |
128 |     InvalidData = sys::godot_error_GODOT_ERR_INVALID_DATA,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:129:24
    |
129 |     InvalidParameter = sys::godot_error_GODOT_ERR_INVALID_PARAMETER,
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:130:21
    |
130 |     AlreadyExists = sys::godot_error_GODOT_ERR_ALREADY_EXISTS,
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:131:20
    |
131 |     DoesNotExist = sys::godot_error_GODOT_ERR_DOES_NOT_EXIST,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:132:24
    |
132 |     DatabaseCantRead = sys::godot_error_GODOT_ERR_DATABASE_CANT_READ,
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:133:25
    |
133 |     DatabaseCantWrite = sys::godot_error_GODOT_ERR_DATABASE_CANT_WRITE,
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:134:25
    |
134 |     CompilationFailed = sys::godot_error_GODOT_ERR_COMPILATION_FAILED,
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:135:22
    |
135 |     MethodNotFound = sys::godot_error_GODOT_ERR_METHOD_NOT_FOUND,
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:136:18
    |
136 |     LinkFailed = sys::godot_error_GODOT_ERR_LINK_FAILED,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:137:20
    |
137 |     ScriptFailed = sys::godot_error_GODOT_ERR_SCRIPT_FAILED,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:138:18
    |
138 |     CyclicLink = sys::godot_error_GODOT_ERR_CYCLIC_LINK,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:139:26
    |
139 |     InvalidDeclaration = sys::godot_error_GODOT_ERR_INVALID_DECLARATION,
    |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:140:23
    |
140 |     DuplicateSymbol = sys::godot_error_GODOT_ERR_DUPLICATE_SYMBOL,
    |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:141:18
    |
141 |     ParseError = sys::godot_error_GODOT_ERR_PARSE_ERROR,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:142:12
    |
142 |     Busy = sys::godot_error_GODOT_ERR_BUSY,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:143:12
    |
143 |     Skip = sys::godot_error_GODOT_ERR_SKIP,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:144:12
    |
144 |     Help = sys::godot_error_GODOT_ERR_HELP,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:145:11
    |
145 |     Bug = sys::godot_error_GODOT_ERR_BUG,
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:146:21
    |
146 |     PrinterOnFire = sys::godot_error_GODOT_ERR_PRINTER_ON_FIRE,
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:152:9
    |
152 |     X = sys::godot_vector3_axis_GODOT_VECTOR3_AXIS_X,
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:153:9
    |
153 |     Y = sys::godot_vector3_axis_GODOT_VECTOR3_AXIS_Y,
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\lib.rs:154:9
    |
154 |     Z = sys::godot_vector3_axis_GODOT_VECTOR3_AXIS_Z,
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:120:11
    |
120 |     Nil = sys::godot_variant_type_GODOT_VARIANT_TYPE_NIL,
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:121:12
    |
121 |     Bool = sys::godot_variant_type_GODOT_VARIANT_TYPE_BOOL,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:122:11
    |
122 |     I64 = sys::godot_variant_type_GODOT_VARIANT_TYPE_INT,
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:123:11
    |
123 |     F64 = sys::godot_variant_type_GODOT_VARIANT_TYPE_REAL,
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:124:19
    |
124 |     GodotString = sys::godot_variant_type_GODOT_VARIANT_TYPE_STRING,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:125:15
    |
125 |     Vector2 = sys::godot_variant_type_GODOT_VARIANT_TYPE_VECTOR2,
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:126:13
    |
126 |     Rect2 = sys::godot_variant_type_GODOT_VARIANT_TYPE_RECT2,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:127:15
    |
127 |     Vector3 = sys::godot_variant_type_GODOT_VARIANT_TYPE_VECTOR3,
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:128:19
    |
128 |     Transform2D = sys::godot_variant_type_GODOT_VARIANT_TYPE_TRANSFORM2D,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:129:13
    |
129 |     Plane = sys::godot_variant_type_GODOT_VARIANT_TYPE_PLANE,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:130:12
    |
130 |     Quat = sys::godot_variant_type_GODOT_VARIANT_TYPE_QUAT,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:131:12
    |
131 |     Aabb = sys::godot_variant_type_GODOT_VARIANT_TYPE_AABB,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:132:13
    |
132 |     Basis = sys::godot_variant_type_GODOT_VARIANT_TYPE_BASIS,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:133:17
    |
133 |     Transform = sys::godot_variant_type_GODOT_VARIANT_TYPE_TRANSFORM,
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:134:13
    |
134 |     Color = sys::godot_variant_type_GODOT_VARIANT_TYPE_COLOR,
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:135:16
    |
135 |     NodePath = sys::godot_variant_type_GODOT_VARIANT_TYPE_NODE_PATH,
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:136:11
    |
136 |     Rid = sys::godot_variant_type_GODOT_VARIANT_TYPE_RID,
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:137:14
    |
137 |     Object = sys::godot_variant_type_GODOT_VARIANT_TYPE_OBJECT,
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:138:18
    |
138 |     Dictionary = sys::godot_variant_type_GODOT_VARIANT_TYPE_DICTIONARY,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:139:20
    |
139 |     VariantArray = sys::godot_variant_type_GODOT_VARIANT_TYPE_ARRAY,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:140:17
    |
140 |     ByteArray = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_BYTE_ARRAY,
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:141:18
    |
141 |     Int32Array = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_INT_ARRAY,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:142:20
    |
142 |     Float32Array = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_REAL_ARRAY,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:143:19
    |
143 |     StringArray = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_STRING_ARRAY,
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:144:20
    |
144 |     Vector2Array = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_VECTOR2_ARRAY,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:145:20
    |
145 |     Vector3Array = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_VECTOR3_ARRAY,
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error[E0308]: mismatched types
   --> core\src\variant.rs:146:18
    |
146 |     ColorArray = sys::godot_variant_type_GODOT_VARIANT_TYPE_POOL_COLOR_ARRAY,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected u32, found i32

error: aborting due to 78 previous errors
```

--- a/README.md
+++ b/README.md
@@ -60,11 +60,18 @@ Starting with an empty Godot project, a `cargo` project can be created inside th
 cargo init --lib
 ```
 
-To use the GDNative bindings in your project you have to add the `gdnative` crate as a dependency.
+To use the GDNative bindings in your project you have to add the `gdnative` crate as a dependency. The crates.io version of the bindings is very out of date, and it's currently preferred to use the `master` branch instead until a new release is made.
 
 ```toml
 [dependencies]
-gdnative = "0.5.0"
+gdnative = { git = "git://github.com/GodotNativeTools/godot-rust" }
+```
+
+You may also vendor the bindings as a local dependency:
+
+```toml
+[dependencies]
+gdnative = { path = "../godot-rust" }
 ```
 
 Since GDNative can only use C-compatible dynamic libraries, the crate type has to be set accordingly.

